libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -21,7 +24,7 @@
       </div>
       <div class="book--content">
         <div class="book--content--cover">
-          <img src="{{ b.coverUrl }}" />
+          <img src="{{ b.coverUrl }}" alt="Book Cover"/>
```

Comment:
This isn't really necessary -> cf. https://www.w3.org/WAI/tutorials/images/decision-tree/

Reply:
Made the appropriate change